shared/estate.js

```diff
@@ -37,7 +37,9 @@ export function calculateMapProps(parcels, size = 20) {
   const x = xs[parseInt(xs.length / 2, 10)]
   const y = ys[parseInt(ys.length / 2, 10)]
   const center = { x, y }
-  const zoom = 1 / (xs.length + ys.length) * 7.5
+  const huge = 1 / (xs.length + ys.length) * 7.5 // only used for big ass estates
```

Comment:
hahahaha 🍑 

What do you think of using a constant to explain what is  `7.5` to avoid magic numbers?

Reply:
to be honest I don't know what that 7.5 comes from :P I didn't write that part, but that works fine for big estates (using the formula from the line below fails to fit some estates that are way wider than taller, or vice versa). i ll add a constant for it but i dunno how to properly name it, zoom multiplier?